recipes/fwdpp/0.5.3/meta.yaml

```diff
@@ -14,19 +14,22 @@ requirements:
     #to get the examples/unit tests
     #compiled.
     - gcc  
-    - zlib
+    - zlib {{CONDA_ZLIB}}*
     - gsl {{CONDA_GSL}}*
```

Comment:
build-only dep doesn't need pinning (doesn't hurt though)

Reply:
I think this its more save in the long run to pin these things. No?

Reply:
If the requirements are really just build-only, they should at most be statically linked. So, for compatibility with other packages it shouldn't really matter. It just means you'll likely pick up those packages in the next `bulk` update -- so you'll get new builds with updated static dependencies which isn't a bad thing, just not always necessarily needed.
I'm rather indifferent on this topic; IMHO it doesn't hurt but also (case-dependent) doesn't have to add any benefits. Hence, if you want to keep it, fine by me.